src/index.html

```diff
@@ -8,7 +8,9 @@
   <title>Colored blocks</title>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
-  <h1>Colored blocks</h1>
+<body class = "full">
```

Comment:
no need to insert spaces around `=` in html

Reply:
there is no need to add class name for `body`, it's only one in the whole document